config/config.go

```diff
@@ -53,6 +51,24 @@ func bindFlags(flagSet *pflag.FlagSet) error {
 	return nil
 }
 
+// Resolve ethereum network based on the set boolean flags.
+func (c *Config) resolveEthereumNetwork(flagSet *pflag.FlagSet) error {
+	var err error
+
+	c.Ethereum.Network, err = func() (commonEthereum.Network, error) {
+		if ok, err := flagSet.GetBool(commonEthereum.Goerli.String()); ok {
+			return commonEthereum.Goerli, err
```

Comment:
I think if we're inside of this `if` then the `err` is nil. What happens when the error occurs? Do we panic?
Same comment for the `if` below

Reply:
36508650cb2a80b8c4b82440a277adc2beadaaf7